geocruncher/main.py

```diff
@@ -20,10 +20,12 @@ def run_geocruncher(args):
     if args[1] == 'tunnel_meshes':
         with open(args[2]) as f:
             data = json.load(f)
+        # sub tunnel are a bit bigger to wrap main tunnel
+        subT = data["idxStart"] != -1 and data["idxEnd"] != -1
```

Comment:
I think the `* 1.05 if subT else ...`  repetiton could be replaced if we instead use something like: 
`subT= 1.05 if (data["idxStart"] != -1 and data["idxEnd"] != -1) else 1.0`